docs/loadingoverlay/customimageoverlay.py

```diff
@@ -0,0 +1,50 @@
+
+import dash_mantine_components as dmc
+import random
+
+# Generate random data for the BarChart
+data = [
+    {
+        "month": month,
+        "Smartphones": random.randint(50, 300),
+        "Laptops": random.randint(30, 200),
+        "Tablets": random.randint(20, 150),
+    }
+    for month in ["January", "February", "March", "April", "May", "June"]
+]
+
+component = dmc.Box(
+    children=[
+        dmc.Stack(
+            pos="relative",
+            p=10,
+            children=[
+                dmc.LoadingOverlay(
+                    visible=True,
+                    id="custom-loading-overlay",
+                    style={'zIndex': 10},
```

Comment:
The `zIndex`is a prop for this component, so you can just do it like this:

 `zIndex=10`